guides/v2.2/cloud/release-notes/cloud-tools.md

```diff
@@ -44,7 +44,7 @@ The release notes include:
 
     -   {:.fix}<!-- MAGECLOUD-3369 -->Fixed an issue that caused Docker deploy to fail with a `Name or service not known` error if the cache is configured for a service that is not available. Now, you can remove a service from the [`.magento/services.yaml` file]({{page.baseurl}}/cloud/project/project-conf-files_services.html). The Docker configuration generator updates the service in the `docker/config.php.dist` file automatically.
 
-    -   {:.new}<!-- MAGECLOUD-3251 -->—Added interactive validations for service compatibility. Now, if a requested service is incompatible with the Magento version or other services, the _interactive mode_ prompts the user with a message and a choice to continue. See the [Service versions]({{page.baseurl}}/cloud/docker/docker-config.html#service-versions) available for Docker. Use the `-n` option to skip the interactivity for CICD purposes.
+    -   {:.new}<!-- MAGECLOUD-3251 -->Added interactive validations for service compatibility. Now, if a requested service is incompatible with the Magento version or other services, the _interactive mode_ prompts the user with a message and a choice to continue. See the [Service versions]({{page.baseurl}}/cloud/docker/docker-config.html#service-versions) available for Docker. Use the `-n` option to skip the interactivity for CICD purposes.
```

Comment:
Just curious--what if they disable interactive mode?  Would that not be recommended?

Reply:
You can disable interactive on any of the CLI commands. It's useful when you don't want to be prompted. If no one is available to answer the prompt, it is of course "hanging" until it receives an answer. LOL not good for automated situation.